GUI/Forms/ExtractProgressForm.cs

```diff
@@ -34,6 +38,11 @@ public ExtractProgressForm(Package package, TreeNode root, string path, bool dec
             this.root = root;
             this.path = path;
             this.decompile = decompile;
+
+            gltfExporter = new GltfModelExporter()
+            {
+                FileLoader = new BasicVpkFileLoader(package)
```

Comment:
Can't `AdvancedGuiFileLoader` be used in GUI? I think it should be pass in the `VrfGuiContext`.

We have `ExportData` class already, perhaps it can be reused.

Reply:
Not sure. The rest of the function only deals with the current package.

This all needs to be moved to `GUI/Types/Exporter/ExportFolder.cs` and then something like `ExportData` could be used to clarify if the folder needs to be GltfExported or just Extract.

Currently `ExtractProgressForm.ExtractFilesAsync` exports supported files to gltf and extracts the rest.